main/pipeline/pipeline.cc

```diff
@@ -156,7 +156,7 @@ unique_ptr<parser::Node> runPrismParser(core::GlobalState &gs, core::FileRef fil
         return std::unique_ptr<parser::Node>();
     }
 
-    return Prism::Translator(parser, gs).translate(std::move(root));
+    return Prism::Translator(std::move(parser), gs).translate(std::move(root));
```

Comment:
This just saves one reference count bump on the `shared_ptr` within the `parser`. Won't someone think of those poor poor clock cycles?!